Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2148,17 +2151,23 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             return WI.Rect.rectFromClientRect(this._elementForBadgeType.get(WI.DOMTreeElement.BadgeType.Event).getBoundingClientRect()).pad(2);
         };
 
-        let popover = new WI.Popover(this);
-        popover.windowResizeHandler = function(event) {
-            popover.present(calculateTargetFrame(), preferredEdges, {updateContent: true, shouldAnimate: false});
+        let delegate = {
```

Comment:
why are we creating this one-off object instead of just using `this` (and putting `didDismissPopover` as a method of `WI.DOMTreeElement`)?

Reply:
Co-location of related logic, really; to make it easier to understand. 
But I can move `didDismissPopover` it back out and pass `this` as the delegate as we do in all other places.

One of these days I'll subclass `Popover` and have this done internally, then update all popover instances.
The ping-pong of self-dismissing popovers and clearing their instance from the outside is odd.